backend/data/src/main/resources/db/changelog/schema.xml

```diff
@@ -3285,15 +3297,18 @@
             <column name="updated_at" type="TIMESTAMP(6) WITHOUT TIME ZONE">
                 <constraints nullable="false"/>
             </column>
+            <column name="created_key_blocks" type="BOOLEAN">
+                <constraints nullable="false"/>
+            </column>
             <column name="key_id" type="BIGINT">
                 <constraints nullable="false"/>
             </column>
-            <column name="lang_tags" type="JSONB"/>
-            <column name="message_ts" type="VARCHAR(255)"/>
+            <column name="language_tags" type="JSONB"/>
+            <column name="message_time_stamp" type="VARCHAR(255)"/>
```

Comment:
This should be message_timestamp. You probably named the backing property `timeStamp`, which is incorrect.